crates/core/src/err/mod.rs

```diff
@@ -915,6 +929,12 @@ pub enum Error {
 		name: String,
 	},
 
+	/// The requested api already exists
+	#[error("The bucket '{value}' already exists")]
+	BuAlreadyExists {
```

Comment:
```suggestion
	BucketAlreadyExists {
```

Reply:
All other errors already follow this format, I'd rather keep it aligned and fully change it in a followup PR...